android/java/org/chromium/chrome/browser/app/domain/WalletModel.java

```diff
@@ -232,29 +243,6 @@ public void updateCoinType() {
             mCryptoModel.updateCoinType();
         }
 
-        /**
-         * The network, coin, and account needs to be update whenever any of the other two are
-         * changed
-         *
-         * @param coin of current account and network, SOL, ETH etc.
-         */
-        @Override
-        public void updateCoinAccountNetworkInfo(@CoinType.EnumType int coin) {
-            if (mBraveWalletService == null) {
-                return;
-            }
-            // Update coin
-            mBraveWalletService.setSelectedCoin(coin);
-            mCryptoModel.updateCoinType(coin, isCoinUpdated -> {
-                // Update account per selected coin
-                mKeyringModel.update();
-                // Update network per selected coin
-                mCryptoModel.getNetworkModel().init();
-                // Update transactions
-                mCryptoModel.refreshTransactions();
-            });
-        }
-
```

Comment:
This was handler for chain_id changed event which could imply account change. Now chain changed only for current account, so no need in these complexities